jasmine.json

```diff
@@ -7,5 +7,5 @@
     "helpers/**/*.js"
   ],
   "stopSpecOnExpectationFailure": false,
-  "random": true
+  "random": false
```

Comment:
Are we intentionally avoiding semi-randomization of jasmine based tests?

Reply:
We were. Singular and its tests use global DOM functions, which leads to test pollution, which makes it hard to write tests that pass when run in random order.

We'll re-write the tests to dependency inject a window variable, instead of using the global. When not passed in, we'll default to the global. This will make the tests more deterministic, and all the tests to continue to be run in random order.

Reply:
We were able to restore test randomization with less drastic changes than what I proposed in my previous comment. The required changes were, before each test run:
1. We needed to unset `window.parent.Singular`.
1. When testing the use case when appendFramesOnInit is not `true`, we need to explicitly update the value.

These changes avoid the test pollution that's observable when running the tests in random order.